tests/ctest/test_elektra.c

```diff
@@ -0,0 +1,82 @@
+/**
+ * @file
+ *
+ * @brief
+ *
+ * @copyright BSD License (see doc/LICENSE.md or http://www.libelektra.org)
+ */
+
+#include <tests_internal.h>
+#include <elektra.h>
+
+static void set (const char * parentKeyName, const char * name, const char * value) {
+    // Open
+    KeySet * config = ksNew (0, KS_END);
+    Key * parentKey = keyNew (parentKeyName, KEY_END);
+    KDB * handle = kdbOpen (parentKey);
+    kdbGet (handle, config, parentKey);
+
+    printf ("Error occurred: %s\n", keyString (keyGetMeta (parentKey, "error/description")));
+
+    // Set
+    Key *key = keyNew(parentKeyName, KEY_END);
+    keyAddName(key, name);
+    keySetString(key, value);
+    ksAppendKey(config, key);
+
+    // Check
+    Key *checkKey = ksLookupByName (config, name, 0);
```

Comment:
This file was never meant to be added to git, I tried different things until I found out that the KDB could not be loaded in the first place. I'm aware of the inconsistencies, please ignore them, the code will get removed anyway after we have found out why the resolver cannot be loaded (or about the real cause).